src/Particular.Analyzers/DiagnosticDescriptors.cs

```diff
@@ -179,5 +179,13 @@ public static class DiagnosticDescriptors
             category: "Code",
             defaultSeverity: DiagnosticSeverity.Warning,
             isEnabledByDefault: true);
+
+        public static readonly DiagnosticDescriptor UseUtcNow = new DiagnosticDescriptor(
+            id: DiagnosticIds.UseUtcNow,
+            title: "Use DateTime.UtcNow or DateTimeOffset.UtcNow",
```

Comment:
```suggestion
            title: "Use DateTime.UtcNow or DateTimeOffset.UtcNow is not used for presenting current time but storing a timestamp.",
```

Reply:
That doesn't make sense.

Reply:
@DavidBoike and now?

Reply:
I'm not sure we need to call this out. In our platform, displaying the current time in a UI is an edge case and the analyzer can be supressed in that case.

But if you feel strong about it:

```suggestion
            title: "Use DateTime.UtcNow or DateTimeOffset.UtcNow unless unless the value is being used for displaying the current date-time in a user's local time zone.",
```

Reply:
@adamralph not strongly about it, I already approved it :-) just a minor improvement to state why usage of (the only reason) for which `.Now` is allowed.